server/pages/index.js

```diff
@@ -119,7 +122,19 @@ function getCurrentCommitShortChecksum() {
 	}
 }
 
-function getDefaultContext( request ) {
+function getFileContentsSafely( filepath ) {
+	return new Promise( function( resolve ) {
+		fs.readFile( filepath, function( err, content ) {
```

Comment:
Maybe it's worth to cache the results in local vars, since the contents of those files aren't gonna change during the lifetime of the server (will they?).